fire/api/firedb.py

```diff
@@ -634,6 +635,98 @@ def basedir_materiale(self):
         konf = self._hent_konfiguration()
         return konf.dir_materiale
 
+    def tilknyt_landsnumre(self, punkter: List[Punkt]) -> List[PunktInformation]:
```

Comment:
Der er lidt mere i det end der umiddelbart kan læses ud fra punkterne, så der skal lige et `punkttype`-argument (enum) med også. Lige nu behøver vi kun typerne GI (1-10, 801-nnnn), MV (11-800) og Højde (9000-nnnn). Der er andre serier i brug, men de er ikke hårdt etablerede. I "minimum viable" versionen er det kun højdefikspunkter der er relevante, men det vil være smart at få de andre to typer (eller i hvert fald GI) med, alene for at markere at der er mere end en nummerserie at bekymre sig om.

Reply:
>  GI (1-10, 801-nnnn), MV (11-800) og Højde (9000-nnnn)

Først og fremmest, etablerer vi nye GI og MV punkter? Derudover, GI serien løber vel fra 1-10 og 801-8999? Ellers kan man ikke skælne om et punkt med løbenummer 9001 er et GI-punkt eller et højdefikspunkt.

Reply:
> Derudover, GI serien løber vel fra 1-10 og 801-8999?

Ja principielt (og formodentlig også i praksis), men der er, iflg Lars, ikke nogen formel øvre grænse. Højdefikspunkterne er principielt også 9000-9999 (eller sandsynligvis 9001-9999), men der kan forekomme løbenumre over 10000.

MV , matrikelpunkter, laver vi ikke flere af, men hvad GI angår, så er det jo fx 5D-nettet der spiller en tilsvarende rolle i GNSS-tidsalderen.

Reply:
Man kunne eventuelt kalde GI-nummerserien for GE (for generisk geodæsi), og højdenummerserien for KO (for koter), Jessennummerserien for JE. Der er også en hjælpepunktnummerserie, som jeg ikke kender udstrækningen af - HJ? og sikkert også en for tyngdemålepunkter TY?

Resten bliver så til MV, som kan omfortolkes fra "Matrikulsvæsenet" til "Med Videre" :-)

Lige nu behøver vi ikke andet end GE, KO (JE er så specielle at de alligevel skal håndteres separat)

Reply:
Okay, men det går vist ikke uden en formel øvre grænse så sådan en vil jeg indføre på GI-numrene. Jeg synes det er fjollet at bygge funktionalitet ind til håndtering af MV-punkter når vi nu ikke gør den slags mere. Vi kan evt. overveje at bruge løbenumrene 10-800 på noget andet?

Er 1-10 i øvrigt ikke vandstandsmålere? Eller husker jeg forkert her.

Reply:
Vandstandsmålerne hedder vist noget med V-xxx (endnu et specialtilfælde, som vi kommer til at patche)

Reply:
Jeg har også en vag erindring om at @majbrittws har nævnt noget om en 19000 højdefikspunktsserie. Men om det måske så er den hjælpepunktsserie Lars snakkede om er ikke lige til...

Reply:
Rettelse: Vandstandsmålere ser sådan ud:

`K -41- V.5` rt     55 17 15.944 sx   10 49 35.115 sx      0.854 m      2001 07 02, 10.46

`K -41- V.6` rt     55 17 16.242 sx   10 49 36.829 sx      0.880 m      2001 10 04, 13.25

`K -41- V.7` rt     55 17 15.944 sx   10 49 35.115 sx      0.881 m      2001 06 26, 11.04

`K -41- V.8` rt     55 17 16.235 sx   10 49 35.128 sx      0.872 m      2005 03 17, 10.07

(søgning omkring Slipshavn med [Gamle Valdemar](http://valdemar.kms.dk/valdemar/index_classic.html))

Reply:
Jeg har forsøgt mig med en løsning i 501cc6f. Umiddelbart ser det ud til at fungere, men mine tests er ikke fyldestgørende. Det kræver et mere omfattende testdatasæt end vi har i øjeblikket. 

Jeg vil godt lige tjekke op på løbenummerintervallerne du har givet her, da jeg synes grænserne er lidt sjove. Fx er det mærkeligt at MV-intervallet slutter på 800 mens Jessen intervallet slutter på 81999. Det betyder nok ikke alverden i praksis, men vi kan lige så godt gøre det helt rigtigt nu vi er i gang...